src/ray/core_worker/core_worker.cc

```diff
@@ -161,15 +161,14 @@ CoreWorkerProcess::CoreWorkerProcess(const CoreWorkerOptions &options)
   // RayConfig is generated in Java_io_ray_runtime_RayNativeRuntime_nativeInitialize
   // for java worker or in constructor of CoreWorker for python worker.
   ray::stats::Init(global_tags, options_.metrics_agent_port);
+
+  // Make sure CoreWorkerProcess is destructed before destructing other static variables
+  // (e.g. spdlog).
```

Comment:
Please add a comment about why this needs to be put in the last.

Reply:
done.